### DISCUSSION TITLE
[Suggestion] Add the infomation for the latest keyboard cache location in MSTG-STORAGE-5

### DISCUSSION THREAD

**Discussion by sohsatoh** [OWASP/mastg#2367] [Ideas]
The "Finding Sensitive Data in the Keyboard Cache" section of MSTG-STORAGE-5 describes the following.

```
Several options for simplifying keyboard input are available to users. These options include autocorrection and spell checking. cached by default, in /private/var/mobile/Library/Keyboard/dynamic-text.dat.
```

However, in iOS14 and iOS16, I confirmed that the keyboard cache is stored in `/private/var/mobile/Library/Keyboard/<lang>-dynamic.lm/dynamic-lexicon.dat`, and the information written in MSTG-STORAGE-5 is already outdated.

**Comment by cpholguera** [comment]
Covered now in https://github.com/OWASP/owasp-mastg/issues/2368

Thank you @sohsatoh!
